app/controllers/mixins/ems_common_angular.rb

```diff
@@ -442,9 +443,9 @@ def build_credentials(ems, mode)
         session[:oauth_response] = nil
       end
       if ems.kind_of?(ManageIQ::Providers::ContainerManager) &&
-         ems.supports_authentication?(:bearer) && params[:bearer_password]
-        creds[:hawkular] = {:auth_key => params[:bearer_password], :userid => "_"}
-        creds[:bearer] = {:auth_key => params[:bearer_password]}
+         ems.supports_authentication?(:bearer) && params[:default_password]
+        creds[:hawkular] = {:auth_key => params[:default_password], :userid => "_"}
+        creds[:bearer] = {:auth_key => params[:default_password]}
```

Comment:
p.s.
`params[:cred_type]` is the form's equivalent of `auth_type/role` in `connection_configurations` ruby method. https://github.com/ManageIQ/manageiq/pull/9235 makes it return "default" instead of "bearer".